diff --git a/scull/scull_load b/scull/scull_load
index 2ef6a54..c70e82f 100755
--- a/scull/scull_load
+++ b/scull/scull_load
@@ -13,7 +13,7 @@ fi
 
 # invoke insmod with all arguments we got
 # and use a pathname, as insmod doesn't look in . by default
-insmod ./$module.ko $* || exit 1
+modprobe $module $* || exit 1
 
 # retrieve major number
 major=$(awk "\$2==\"$module\" {print \$1}" /proc/devices)
